packages/devtools_app/lib/src/inspector/inspector_screen.dart

```diff
@@ -199,18 +201,11 @@ class InspectorScreenBodyState extends State<InspectorScreenBody>
   List<Widget> getServiceExtensionWidgets() {
     return [
       ServiceExtensionButtonGroup(
-        minIncludeTextWidth: 1050,
-        extensions: [extensions.slowAnimations],
-      ),
-      const SizedBox(width: denseSpacing),
-      ServiceExtensionButtonGroup(
-        minIncludeTextWidth: 1050,
-        extensions: [extensions.debugPaint, extensions.debugPaintBaselines],
-      ),
-      const SizedBox(width: denseSpacing),
-      ServiceExtensionButtonGroup(
-        minIncludeTextWidth: 1250,
+        minIncludeTextWidth: 1150,
```

Comment:
put this in a const above `serviceExtensionButtonsIncludeTextWith`

Reply:
done